### PR TITLE
[Spree Upgrade] Remove dependency to spree frontend - views and assets

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

Starts #3676
I was exploring the idea of making OFN independent of Spree, I started with some simple things like making OFN independent from spree_frontend code. This PR makes OFN independent of Spree Frontend app/views and app/assets folders.

Note on the GA script: I didnt convert to haml because, now that matomo is serving us well, I think we will soon drop support for GA, so this file will just be deleted.

#### What should we test?
- We need to review signup process to make sure the dead code deletion is not creating any issues. Basically making sure that all errors in the signup page don't redirect the user but just show what's wrong (for example passwords dont match).
- Verify the small spinner gif shows up where it was showing up before, it's the gif that shows up in the "Updating cart..."/"Edit your cart" button when changing quantity of products in the product list (frontoffice). If we test it shows up there, I think we will be fine.
- We could test the GA integration... is it worth it?

#### How is this related to the Spree upgrade?
This is part of the end-of-spree-upgrade-clean-up...

### CODE REVIEW

File: app/controllers/user_registrations_controller.rb [hunk 1]
```diff
@@ -39,9 +39,6 @@ def create
   def render_error(errors = {})
     clean_up_passwords(resource)
     respond_to do |format|
-      format.html do
-        render :new
-      end
```
[the remark this answers is not in the document]
Comment (luisramos0) [reply app/controllers/user_registrations_controller.rb]: no, it's always ajax from the signup modal. I have tested now in the checkout process and confirmed it.
I added a new commit to remove the html part of the main request.

**Comment by sauloperez** [comment]
@luisramos0 do you want to answer before we merge?

**Comment by luisramos0** [comment]
yes, I think it's ok to skip GA tests.

personally I dont like that spinner in the button. shall we create another issue/PR to remove the spinner?

**Comment by RachL** [comment]
@luisramos0 fine by me. But are you okay if we keep the spinner on the Update button? Or if we enable the button can we make a larger page spinner so users can know something is loading (they don't look at the browser spinner)?

**Comment by luisramos0** [comment]
yes, I am ok with keeping or removing the spinner.

that means we merge this PR as is @RachL ?

**Comment by luisramos0** [comment]
ah, I see you moved to ready to go. I'll merge.

**Pull request merged**